unicodetools/src/main/resources/org/unicode/props/IndexUnicodeProperties.txt

```diff
@@ -310,6 +313,15 @@ Unihan_Variants ; kSpoofingVariant
 Unihan_Variants ; kTraditionalVariant
 Unihan_Variants ; kZVariant
```

Comment:
Should there be a line for kZhuang here? (In other words, are you getting any data for kZhuang?)